changelog_entries/wc-double-scrollbar-bugfix.md

```diff
@@ -0,0 +1,2 @@
+### Multiplayer
```

Comment:
WC is normally under `### Campaigns` `    * World Conquest` - copy an entry from changelog.md as a starting point.